dist/build_deploy.sh

```diff
@@ -15,6 +15,7 @@ function cleanup() {
     set +e
     if [[ ! -n "$KEEP_RELEASE_OUTPUT" ]]; then
         rm -f ${RELEASE_OUTPUT_DIR}/{graph-builder,policy-engine}
+        rm -f ${DOCKERFILE_DEPLOY}
```

Comment:
this will remove original dockerfile :)
I think it should be `${RELEASE_OUTPUT_DIR}/Dockerfile`